package.json

```
@@ -9,7 +9,11 @@
     "lint:eslint": "eslint . --fix",
     "lint": "npm-run-all prettier lint:*",
     "prettier": "prettier --write .",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "start:webpack": "webpack --mode=development --watch",
+    "start:server": "nodemon index.js --watch dist/server.js",
+    "start-ssr": "NODE_ENV=development npm-run-all --print-label --parallel start:*",
```

🧐
Обычно нет причин запускаться без ssr, если его затащили
```suggestion
    "start": "NODE_ENV=development npm-run-all --print-label --parallel start:*",
```

---

Согласен